boofuzz/primitives/bit_field.py

```diff
@@ -106,8 +106,20 @@ def _iterate_fuzz_lib(self):
                 self.max_num // 32,
                 self.max_num,
             ]
+
+            # To avoid duplication of mutation params, we introduce gradually
+            # increasing lower border. We also require interesting borders to be
+            # sorted in ascending order to be processed correctly.
+            # Deduplication could also be done with intermediate 'set' construction,
+            # but it might be costly or impossible if the number of values is huge.
+            interesting_boundaries.sort()
```

Comment:
I'd suggest moving the initialization of `interesting_boundaries` as well as the sorting of it into the init function as a private member variable.

`self.max_num` will never change so there is no point in re-calculating the boundaries for every mutation.

Reply:
Hi @SR4ven !
Thanks for the feedback, will apply it and push